cms/djangoapps/contentstore/views/item.py

```diff
@@ -294,6 +294,7 @@ def xblock_view_handler(request, usage_key_string, view_name):
 
         elif view_name in PREVIEW_VIEWS + container_views:
             is_pages_view = view_name == STUDENT_VIEW   # Only the "Pages" view uses student view in Studio
+            is_library_view = view_name == 'container_child_preview'
```

Comment:
@cahrens I updated the variable name in c777d8393ea4bd802267d25d75de25bcba7bef23, please have another look.